dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -242,6 +242,9 @@ class ApiAccessor {
                 testDataProvider = testDataProviderForP2pData,
                 frameworkDataUploadFunction = this::p2pUploaderFunction,
             )
+            else -> {
```

Comment:
Why was this necessary?

Reply:
datatype Enum hat GDV dazubekommen, der Accessor ist aber nicht in der Lage GDV daten hochzuladen

Reply:
Genau.  Für das `when` statement sind nicht alle möglichen Fälle abefangen, weil der Fall "gdv" nicht explizit abgefangen wird.
Deswegen ist man gezwungen, ein `else` für alle nicht abefangenen Fälle einzuführen.
Gleichzeitig soll der api-accessor jedoch für die `else` Fälle eben nicht benutzt werden. Deswegen soll in dem Fall sofort ein error geschmissen werden.

Reply:
ok